workflows/prognostic_c48_run/docs/config-usage.rst

```diff
@@ -63,6 +63,15 @@ Notice how these configurations correspond with
 :py:attr:`runtime.config.UserConfig.nudging`. Refer to those docs as a
 reference.
 
+In addition to the above settings, to prescribe the sea surface temperatures to
+the values in the reference restart files, the
+``gfs_physics_nml.use_climatological_sst`` namelist parameter must be set to
+``false``::
+
+    namelist:
+        gfs_physics_nml:
+            use_climatological_sst: false
```

Comment:
Does this also need to be set for making nudge-to-obs runs follow the analysis SSTs? I know the `use_analysis_sst` flag exists and needs to be set appropriately for those runs, but not sure how it interacts with this flag.

Reply:
It's a good question.

This flag, as currently written, toggles between the behavior before or after VulcanClimateModeling/fv3gfs-fortran#93.  It is possible that VulcanClimateModeling/fv3gfs-fortran#93 broke setting the SST to the analysis via the `use_analysis_sst` flag too, in which case you would want to set `use_climatological_sst = .false.` in those cases as well.  I'm currently running a couple tests to see.

Reply:
It's possible the flags act as "use_climatological_sst: use prescribed sst", and "use_analysis_sst: prescribed sst is climatology (false) or analysis (true)". Does use_analysis_sst make it so `tsfco` is the analysis sst?

Reply:
> Does use_analysis_sst make it so `tsfco` is the analysis sst?

Yes, this is why I am concerned.

Reply:
This turned out to basically be a non-issue; see analysis for the nudged-to-obs case in [this notebook](https://github.com/VulcanClimateModeling/explore/blob/master/spencerc/2021-03-23-test-sst-update/2021-03-25-sst-setting-n2o.ipynb).  When using `use_analysis_sst = .true.` both `Sfcprop%tsfco` and `Sfcprop%tsfc` are set to the analysis values.

*Technically* I think the most correct approach would be to set the new `use_climatological_sst` flag to `.false.` and only set the value of `Sfcprop%tsfco`; however, one can still achieve almost the same results leaving `use_climatological_sst = .true.` and setting both `Sfcprop%tsfco` and `Sfcprop%tsfc` (the results are not bitwise identical, but are close enough).  My understanding is that the surface cycling in the fortran model is weak, and only acts to slowly nudge `Sfcprop%tsfc` toward the climatology; if it is constantly overridden, then it stays very close to its overridden value.

Reply:
Two years later in refreshing myself on the discussion in this thread, I believe I linked to the wrong notebook in the above comment.  [This one](https://github.com/ai2cm/explore/blob/master/spencerc/2021-03-23-test-sst-update/2021-03-24-verify-sst-setting-n2o.ipynb) appears to be more polished and relevant to the discussion.